develop/tutorials/articles/194-adaptive-media/00-adaptive-media-images-rest-apis-intro.markdown

```diff
@@ -1,312 +1,365 @@
+Adaptive Media images.
+
+You can learn how to use the */configuration* endpoint next.
+
+## Configuration [](id=configuration)
+
+The following requests are possible through the */configuration* endpoint:
+
+### List All Available Configurations [](id=list-all-available-configurations)
+
+Below is an example request pattern:
+
+    GET [server address]/o/adaptive-media-rest/images/configuration
+
+**Status Codes**
+
+200<br />
+500
+
+Below is an example response object:
+
+    [
+    {
+    "name": "Extra large demo size",
+    "id": "demo-xlarge",
+    "max-width": "1200",
+    "max-height": "1200"
+    },
+    {
+    "name": "Extra small demo size",
+    "id": "demo-xsmall",
+    "max-width": "50",
+    "max-height": "50"
+    },
+    {
+    "name": "Large demo size",
+    "id": "demo-large",
+    "max-width": "800",
+    "max-height": "800"
+    }
+    ]
+
+### Get a Configuration by its ID [](id=get-a-configuration-by-its-id)
+
+Below is an example request pattern:
+
+    GET [server address]/o/adaptive-media-rest/images/[configurationId]
+
+**Status Codes**
+
+200<br />
+404 (Non-existent configuration)
+
+Below is an example response object:
+
+    {
+    "name": "Small demo size",
+    "id": "demo-small",
+    "max-width": "100",
+    "max-height": "100"
+    }
+
+### Add a New Configuration [](id=add-a-new-configuration)
+
+Below is an example request pattern:
+
+    PUT [server address]/o/adaptive-media-rest/images/[configurationId]
+
+**Status Codes**
+
+200<br />
+400 (wrong body)<br />
+403 (user without permission)
+
+Below is an example request:
+
+    PUT [server address]/o/adaptive-media-rest/images/small
+
+    BODY:
+    {
+    "name": "Small Size",
+    "max-height": 100,
+    "max-width": 100
+    }
+
+Here is the resulting configuration:
+
+    {
+    "name": "Small Size",
+    "id": "small",
+    "max-height": "100",
+    "max-width": "100"
+    }
+
+### Delete a Configuration [](id=delete-a-configuration)
+
+Below is an example request pattern:
+
+    DELETE [server address]/o/adaptive-media-rest/images/[configurationId]
+
+**Status Codes**
+
+200<br />
+403 (user without permission)
+
+Below is an example request:
+
+    DELETE [server address]/o/adaptive-media-rest/images/small
+
+    BODY:
+    {
+    "name": "Small Size",
+    "max-height": 100,
+    "max-width": 100
+    }
+
+Here is the resulting deleted configuration:
+
+    {
+    "name": "Small Size",
+    "id": "small",
+    "max-height": "100",
+    "max-width": "100"
+    }
+
+Now that you know how to use the */configuration* endpoint, you can learn how to
+use the */content* endpoint next.
+
+## Content Endpoint [](id=content-endpoint)
+
+To use the */content* endpoint you must first choose the Adaptive Media
+image you want to recover. You can specify an adaptive media image via two
+possible forms of ID: `fileEntryId` or `fileVersionId`.
+<!-- Where can I find this information for the Adaptive Media images? -->
```

Comment:
You will recover this ids by retrieving a list from the portal. Right now, a developer will have to do this using the json-web-services since this is the first REST API.